src/data/services/lms/urls.js

```diff
@@ -22,10 +22,11 @@ export const useViewUrl = () => {
 
 export const usePageDataUrl = (step) => {
   const baseUrl = useBaseUrl();
-  if ([stepNames.submission, stepNames.xblock].includes(step)) {
-    return `${baseUrl}/get_block_learner_submission_data`;
-  }
-  return `${baseUrl}/get_block_learner_assessment_data/${step}`;
+  // if ([stepNames.submission, stepNames.self, stepNames.xblock].includes(step)) {
+  //   return `${baseUrl}/get_learner_data`;
+  // }
+  // return `${baseUrl}/get_learner_data/${step}`;
+  return `${baseUrl}/get_learner_data`;
 };
```

Comment:
Why not passing the step?  is required for the features to work.

Reply:
This probably has to be step 2. It produced more bugs on the backend than we bargain for.

Reply:
for the sake of immediate integration, I propose adding the step *if it is in the accepted list*.